projects/igniteui-angular/src/lib/grids/filtering/base/grid-filtering-row.component.html

```diff
@@ -48,7 +48,7 @@
         [value]="value"
         [outlet]="filteringService.grid.outlet"
         [locale]="filteringService.grid.locale"
-        (onSelection)="onDateSelected($event)"
+        (valueChange)="onDateSelected($event)"
         (onClosed)="datePickerClose()">
         <ng-template igxDatePickerTemplate let-openDialog="openDialog">
```

Comment:
Oh boy, just noticed this isn't the simplest of uses and it's the deprecated template...

Reply:
Will be fixed in a separate PR.